src/components/pacbio/PacbioPoolLibraryEdit.vue

```diff
@@ -145,14 +145,21 @@ export default {
     ...mapMutations(['updateLibrary']),
     ...mapActions(['applyTags']),
     hasErrors(attribute) {
-      if (!this.library.errors) {
-        return null
+      //Check any errors exist in library for the given attribute
```

Comment:
This can be simplified to:
```
hasErrors(attribute) {
  return !this.library?.errors?.[attribute]
}
```
the ? is known as optional chaining which means you don't have to repeat stuff. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Done